cli/src/Types.ts

```diff
@@ -92,19 +93,27 @@ export const AvailableGroups: readonly WorkingGroups[] = [
   WorkingGroups.StorageProviders
 ] as const;
 
+export type Reward = {
+    totalRecieved: Balance;
+    value: Balance;
+    interval: number; // 0 = reward is not really "recurring"
```

Comment:
wouldn't be better to make interval and nextPaymentBlock optional?
```suggestion
    interval?: number;
```

or

```suggestion
    interval: number | undefined;
```

Reply:
Fixed